src/main/io/osd.c

```diff
@@ -1829,6 +1837,14 @@ static void osdShowStats(uint16_t endBatteryVoltage)
     }
 #endif
 
+#ifdef USE_GYRO_DATA_ANALYSE
+    if (osdStatGetState(OSD_STAT_MAX_FFT) && featureIsEnabled(FEATURE_DYNAMIC_FILTER)) { 
+        itoa(stats.max_fft, buff, 10);
+        strcat(buff, "HZ");
+        osdDisplayStatisticLabel(top++, "MAX ROLL FFT", buff);
```

Comment:
Why constrain it to only roll? Wouldn't it make sense to capture the max value across all axes?